spec/graphql/schema/subscription_spec.rb

```diff
@@ -445,13 +457,24 @@ def in_memory_subscription_count
     end
   end
 
+  describe "applying `loads:`" do
+    it "includes `as:` in the event topic" do
+      assert_equal [], SubscriptionFieldSchema::InMemorySubscriptions::EVENT_REGISTRY.keys
+      matz = SubscriptionFieldSchema::USERS["matz"]
+      obj = OpenStruct.new(toot: { body: "I am a C programmer" }, user: matz)
+      SubscriptionFieldSchema.subscriptions.trigger(:toot_was_tooted, {handle: "matz"}, obj)
+      # TODO How does this compare to before-this-change?
+      assert_equal [":tootWasTooted:user:matz"], SubscriptionFieldSchema::InMemorySubscriptions::EVENT_REGISTRY.keys
```

Comment:
This is kind of important: I didn't check exactly how keys were serialized before, and I need to, yknow, _not break them_